Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -735,6 +737,28 @@ def refreshDICOMWidget():
     return True
 
 
+def getEnabledPlugins(self):
+    """Get all available DICOM plugins that were not disabled in application settings.
+    Settings are editable in DICOMPluginSelector widget.
+    """
+
+    # Get list of disabled plugins from application settings
+    disabledPlugins = []
+    settings = qt.QSettings()
+    if settings.contains("DICOM/disabledPlugins/size"):
+        size = settings.beginReadArray("DICOM/disabledPlugins")
+        for i in range(size):
+            settings.setArrayIndex(i)
+            disabledPlugins.append(str(settings.allKeys()[0]))
+        settings.endArray()
+
+    for pluginClass in slicer.modules.dicomPlugins:
+        if pluginClass not in disabledPlugins:
+            enabledPlugins.append(pluginClass)
+
+    return enabledPlugins
+
+
 def getLoadablesFromFileLists(fileLists, pluginClassNames=None, messages=None, progressCallback=None, pluginInstances=None):
```

Comment:
Would make sense to rename `pluginClassNames` parameter to `selectedPlugins`

Reply:
`pluginClassNames` is better in that it specifies that it expects class names (and for example not instances). Even if the new name was considerably better, it may not justify a breaking change (developers may use the argument name for clarity and robustness).

Maybe we should use `pluginClassNames` in the newly added arguments, too, to make it more clear if class names are expected?

Reply:
Definitely makes sense to set the parameter name to `pluginClassNames` for the new changes to `loadSeriesByUID` and `loadByInstanceUID`. That way we also get consistent name at least within the file level.